src/components/Lightbox/index.tsx

```diff
@@ -217,7 +217,7 @@ function Lightbox({isAuthTokenRequired = false, uri, onScaleChanged: onScaleChan
                             >
                                 <Image
                                     source={{uri}}
-                                    style={contentSize ?? DEFAULT_IMAGE_DIMENSION}
+                                    style={[contentSize ?? DEFAULT_IMAGE_DIMENSION, !contentSize && {opacity: 0}]}
```

Comment:
Kindly remove inline styles.

And `contentSize ?? DEFAULT_IMAGE_DIMENSION` why are we still setting DEFAULT_IMAGE_DIMENSION and hiding via opacity.

@tienifr

Reply:
Got it.